manifests/dhparam.pp

```diff
@@ -15,12 +15,13 @@
 #   - `puppetlabs/stdlib`
 #
 define openssl::dhparam(
-  $path,
+  $path = $name,
   $ensure = present,
-  $size = 512,
+  $size = 2048,
```

Comment:
https://security.stackexchange.com/questions/47204/dh-parameters-recommended-size (and this is from 2013) time to increase the default keysize